MIGRATING.md

```diff
@@ -709,7 +722,7 @@ Make sure to update your imports to the new package name and file names:
 
 ## Gotchas
 
-### The new plugins generates missing imports
+### The new plugins generates missing imports <a id="missing-imports"></a>
```

Comment:
There's already a standard anchor for all headlines. It's used by the GH outline feature. Best to stick to the standard anchor.

```suggestion
### The new plugins generates missing imports
```

Reply:
Sounds good. I was just worried that if the wording changed for that heading, the link to it would be broke.